test/heatmap.html

```diff
@@ -19,91 +18,156 @@
 -->
 
 <html>
-    <head>
```

Comment:
Should create a new test case instead of modifying the exists cases.

Reply:
Actually the new case is almost seprate except under same page and share same data. Changes are more likely a false detection (may be due to formatter, actually I didn't change the original case any)